packages/toolbox/src/commands/chainlink/aggregatorsUtils.ts

```diff
@@ -29,35 +17,34 @@ export type Aggregator = {
 const feedMap: Partial<
   Record<CurrencyTypes.EvmChainName, [chainKey: string, networkName: string]>
 > = {
-  mainnet: ['ethereum', 'Ethereum Mainnet'],
-  goerli: ['ethereum', 'Goerli Testnet'],
-  rinkeby: ['ethereum', 'Rinkeby Testnet'],
-  fantom: ['fantom', 'Fantom Mainnet'],
-  matic: ['polygon', 'Polygon Mainnet'],
-  xdai: ['gnosis-chain', 'Gnosis Chain Mainnet'],
-  bsc: ['bnb-chain', 'BNB Chain Mainnet'],
-  avalanche: ['avalanche', 'Avalanche Mainnet'],
-  optimism: ['optimism', 'Optimism Mainnet'],
-  'arbitrum-one': ['arbitrum', 'Arbitrum Mainnet'],
-  moonbeam: ['moonbeam', 'Moonbeam Mainnet'],
+  mainnet: ['mainnet', 'Ethereum Mainnet'],
```

Comment:
FYI this data is now available on the chains API: https://api.request.finance/currency/chains/mainnet, see `priceFeeds`

Reply:
@benjlevesque The values are different with this other API 😕

Reply:
OK. maybe we can update the API values to match that

Reply:
but not needed now